databases/week2/lesson2_homework_part2.sql

```diff
@@ -0,0 +1,40 @@
+SET NAMES utf8mb4;
+
+create database if not exists `school`;
+
+USE school;
+
+
+CREATE TABLE `class` (
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `name` VARCHAR(255) NOT NULL,
+    `begins` DATE NOT NULL,
+    `ends` DATE NULL DEFAULT NULL
+)  ENGINE=INNODB DEFAULT CHARSET=UTF8MB4 COLLATE = UTF8MB4_UNICODE_CI;
+
+CREATE TABLE `student` (
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `name` VARCHAR(255) NOT NULL,
+    `email` VARCHAR(255) NOT NULL,
+    `phone` VARCHAR(255) NULL,
```

Comment:
char lenght is to long. Ideal phone number won't be longer than 15 digits(characters) , same for email & name